index.bs

```diff
@@ -947,6 +950,12 @@ spec:css-syntax-3;
     1.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
+    1.  If |options|[{{CredentialRequestOptions/identity}}] [=map/exists=] then
+        if |settings|' [=relevant global object=]'s [=associated Document=] is **not**
+        [=allowed to use=] the "fedcm"
```

Comment:
Per my comment on the other PR, let's rename the value to something different from `fedcm`.

Reply:
Now that we've settled on a new name on the other PR, I'll update this one as well.